### PR TITLE
FAQ on how stats are calculated.

### PR DESCRIPTION
**Pull request by SushrutHarness**
Thanks for contributing to the Harness Developer Hub! Our code owners will review your submission.

## Description

* Please describe your changes: \__________________________________
* Jira/GitHub Issue numbers (if any): \______________________________
* Preview links/images (Internal contributors only): \__________________

## PR lifecycle

We aim to merge PRs within one week or less, but delays happen sometimes.

If your PR is open longer than two weeks without any human activity, please tag a [code owner](https://github.com/harness/developer-hub/blob/main/.github/CODEOWNERS) in a comment.

PRs must meet these requirements to be merged:

- [ ] Successful preview build.
- [ ] Code owner review.
- [ ] No merge conflicts.
- [ ] Release notes/new features docs: Feature/version released to at least one prod environment.

### CODE REVIEW

File: kb/continuous-delivery/continuous-delivery-faqs.md [hunk 1]
```diff
@@ -6928,3 +6928,17 @@ Both approaches will result in `key2` in the `my_map` variable being set to a `n
 
 #### Why am I getting an error that the input set does not exist in the selected Branch?
 This happens because pipelines and input sets need to exist in the same branch when storing them in Git. For example, if your pipeline exists in the `dev` branch but your input set exists in the `main` branch, then loading the pipeline in the `dev` branch and attempting to load the input set will cause this error. To fix this, please ensure that both the pipeline and input set exist in the same branch and same repository.
+
+#### How are the statistics on the continuous delivery overview page calculated?
```
Comment (douglas-j-bothwell) [inline kb/continuous-delivery/continuous-delivery-faqs.md:6932]:
```suggestion
#### How are the statistics on the Continuous Delivery overview page calculated?
```

File: kb/continuous-delivery/continuous-delivery-faqs.md [hunk 2]
```diff
@@ -6928,3 +6928,17 @@ Both approaches will result in `key2` in the `my_map` variable being set to a `n
...
+
+The [continous delivery overview page](/docs/continuous-delivery/monitor-deployments/monitor-cd-deployments#overview-and-services-dashboards) displays statistics for deloyments and services that can sometimes have numbers that seem unreasonable or incorrect. However, these numbers are correct and are generally caused by the time frame that the statistic is measured over.
```
Comment (douglas-j-bothwell) [inline kb/continuous-delivery/continuous-delivery-faqs.md:6934]:
```suggestion
The [Continous Celivery overview page](/docs/continuous-delivery/monitor-deployments/monitor-cd-deployments#overview-and-services-dashboards) displays statistics for deployments and services that can sometimes seem unreasonable or incorrect. However, these numbers are correct and are generally caused by the time frame that the statistic is measured over. 
```

File: kb/continuous-delivery/continuous-delivery-faqs.md [hunk 3]
```diff
@@ -6928,3 +6928,17 @@ Both approaches will result in `key2` in the `my_map` variable being set to a `n
 
 #### Why am I getting an error that the input set does not exist in the selected Branch?
 This happens because pipelines and input sets need to exist in the same branch when storing them in Git. For example, if your pipeline exists in the `dev` branch but your input set exists in the `main` branch, then loading the pipeline in the `dev` branch and attempting to load the input set will cause this error. To fix this, please ensure that both the pipeline and input set exist in the same branch and same repository.
+
+#### How are the statistics on the continuous delivery overview page calculated?
+
+The [continous delivery overview page](/docs/continuous-delivery/monitor-deployments/monitor-cd-deployments#overview-and-services-dashboards) displays statistics for deloyments and services that can sometimes have numbers that seem unreasonable or incorrect. However, these numbers are correct and are generally caused by the time frame that the statistic is measured over. 
+
+For example, say that we have a statistic `failure rate` that is measured over a 30 period. Then we collect the following data:
```
Comment (douglas-j-bothwell) [inline kb/continuous-delivery/continuous-delivery-faqs.md:6936]:
```suggestion
For example, say that we have a statistic `failure rate` that is measured over 30 days. Then we collect the following data:
```

File: kb/continuous-delivery/continuous-delivery-faqs.md [hunk 4]
```diff
@@ -6928,3 +6928,17 @@ Both approaches will result in `key2` in the `my_map` variable being set to a `n
...
+
+Days 0 - 30 `failure rate`: 5%
+Days 31 - 60 `failure rate`: 15%
+
+Then when displaying the `failure rate` for the last 30 days you would see `15%` and you would see that your `failure rate` increased by `300%`!
+So the statistic will have a red arrow pointing up marked as a `300%` increment in failure rate. 
+
+This value is not incorrect, but measures the change in rate.
```
Comment (douglas-j-bothwell) [inline kb/continuous-delivery/continuous-delivery-faqs.md:6944]:
```suggestion
This value accurately measures the change in the failure rate, from 5% to 15%. 
```

**Review by douglas-j-bothwell — DISMISSED** [review]
A few sugs otherwise LGTM

File: kb/continuous-delivery/continuous-delivery-faqs.md [hunk 5]
```diff
@@ -6928,3 +6928,17 @@ Both approaches will result in `key2` in the `my_map` variable being set to a `n
 
 #### Why am I getting an error that the input set does not exist in the selected Branch?
 This happens because pipelines and input sets need to exist in the same branch when storing them in Git. For example, if your pipeline exists in the `dev` branch but your input set exists in the `main` branch, then loading the pipeline in the `dev` branch and attempting to load the input set will cause this error. To fix this, please ensure that both the pipeline and input set exist in the same branch and same repository.
+
+#### How are the statistics on the Continuous Delivery overview page calculated?
+
+The [Continous Celivery overview page](/docs/continuous-delivery/monitor-deployments/monitor-cd-deployments#overview-and-services-dashboards) displays statistics for deployments and services that can sometimes seem unreasonable or incorrect. However, these numbers are correct and are generally caused by the time frame that the statistic is measured over.
```
Comment (brian-f-harness) [inline kb/continuous-delivery/continuous-delivery-faqs.md:6934]:
```suggestion
The [Continous Delivery overview page](/docs/continuous-delivery/monitor-deployments/monitor-cd-deployments#overview-and-services-dashboards) displays statistics for deployments and services that can sometimes seem unreasonable or incorrect. However, these numbers are correct and are generally caused by the time frame that the statistic is measured over. 
```

**Review by brian-f-harness — APPROVED** [review]

**Review by brian-f-harness — APPROVED** [review]

**Pull request merged**